.github/workflows/call-calc-coverage.yml

```diff
@@ -4,15 +4,14 @@ name: call-calc_coverage
 # The default is to run the workflow on every push or pull request to every branch.
 on:
   workflow_dispatch:
-    paths-ignore:
-      - .devcontainer
-      - .github
-      - 'LICENSE'
-      - 'README.md'
-      - 'CONTRIBUTING.md'
-      - 'Rbuildignore'
-      - '.gitignore'
-      - 'man'
```

Comment:
Shouldn't we continue to ignore these files/folders? It seems to me like we would not care if we have code coverage of the manual files and these other non-code files. Was there something that was not working with these being ignored?

Reply:
Oh yeah, right. I forgot but, still, I think that we can keep these lines so the code coverage is not ran when a PR or push is made only to these files. For example, there is a GitHub action that only changes README.md and we wouldn't want to run the codecoverage test on that PR to dev. Or, if devtools::document() is ran and only changes are made to the man/*.Rd files we wouldn't want to run the coverage.

Reply:
Got it! I just added those lines back without making any changes. I’ll leave the final edits to @k-doering-NOAA through PR #720. Basically I’m not sure where to place `paths-ignore`. Should it go under `workflow-dispatch`, `push`, or `pull`?

Reply:
I think we have to repeat them under both push and pull so that they are all ignored on both PR and pushes. For the workflow dispatch i think maybe we just want to test them all?